pkg/restore/restore.go

```diff
@@ -2243,7 +2325,9 @@ func (ctx *restoreContext) getOrderedResourceCollection(
 				continue
 			}
 
-			if namespace == "" && !boolptr.IsSetToTrue(ctx.restore.Spec.IncludeClusterResources) && !ctx.namespaceIncludesExcludes.IncludeEverything() {
+			if groupResource.Resource == "namespaces" {
+				ctx.log.Infof("Including resource namespaces despite being cluster-scoped")
```

Comment:
is this log message needed?

It can also be confusing if `ctx.restore.Spec.IncludeClusterResources` is set to `true`